airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -850,6 +859,8 @@ def __init__(
         project_id: str | None = None,
         gcp_conn_id: str = "google_cloud_default",
         impersonation_chain: str | Sequence[str] | None = None,
+        deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False),
+        job_poll_interval: float = 10.0,
```

Comment:
I have noticed 2 different values for this parameter in operator and hook: one is 10 seconds, the other one is 2 seconds. Maybe better to have the same value?

Reply:
Updated the parameter in trigger